nss-day-cohort-41/tabloid-fullstack-crop-circle-cavorters
# View Post Details and Create Post

# Description
Developed two features:

 1)  feature that will display the Published Posts.

 2)  feature that displays the details for a given Post. 

# Ticket Reference
> [#5 - View Post Comments](https://github.com/nss-day-cohort-41/tabloid-fullstack-crop-circle-cavorters/issues/5)

> [#6 - Create a Post](https://github.com/nss-day-cohort-41/tabloid-fullstack-crop-circle-cavorters/issues/6)
> 

## Type of change


# Testing Instructions
- Run the API in Visual studio, and `npm serve` the client folder.
- Type `localhost:3000/users` into the browser bar.
- Visually confirm that you're rerouted back to the login page.
- Start program with ISS Express
- Go to localhost:5001 in your browser.
-Login with "admin@example.com"
- Navigate to the Posts link in the navbar.
- Click `View Post` to see the post's detals
- Click `Posts` in the NavBar to go back to the list of posts
- Click the `New Post` button to navigate to the create new post form
- Complete all input fields and click the `Submit` button
- Clicking that button will create a new post a nd navigate you to the detail view for it.


# Checklist:
